api/resolvers/notifications.js

```diff
@@ -316,13 +316,36 @@ export default {
 
       if (meFull.noteCowboyHat) {
         queries.push(
-          `(SELECT id::text, updated_at AS "sortTime", 0 as "earnedSats", 'Streak' AS type
+          `(SELECT id::text, updated_at AS "sortTime", 0 as "earnedSats", 'CowboyHat' AS type
           FROM "Streak"
           WHERE "userId" = $1
           AND updated_at < $2
+          AND type = 'COWBOY_HAT'
           ORDER BY "sortTime" DESC
           LIMIT ${LIMIT})`
         )
+        for (const type of ['HORSE', 'GUN']) {
+          const gqlType = type.charAt(0) + type.slice(1).toLowerCase()
```

Comment:
Will refactor this into a function in a follow-up PR since we use code like this at several places already and I don't want to refactor all these places in this PR.

Reply:
Sounds good. Reviewing this now.